tests/Endpoints/MultiSearchTest.php

```diff
@@ -90,4 +91,38 @@ public function testSupportedQueryParams(): void
         self::assertTrue($result['showRankingScore']);
         self::assertTrue($result['showRankingScoreDetails']);
     }
+
+    public function testMultiSearchWithDistinctAttribute(): void
+    {
+        $response = $this->client->multiSearch([
+            (new SearchQuery())->setIndexUid($this->booksIndex->getUid())
+                ->setQuery('princ')
+                ->setSort(['author:desc']),
+            (new SearchQuery())->setIndexUid($this->booksIndex->getUid())
+                ->setDistinct('genre'),
+        ]);
+
+        self::assertCount(2, $response['results']);
+
+        self::assertArrayHasKey('indexUid', $response['results'][0]);
+        self::assertArrayHasKey('hits', $response['results'][0]);
+        self::assertArrayHasKey('query', $response['results'][0]);
+        self::assertArrayHasKey('limit', $response['results'][0]);
+        self::assertArrayHasKey('offset', $response['results'][0]);
+        self::assertArrayHasKey('estimatedTotalHits', $response['results'][0]);
+        self::assertCount(2, $response['results'][0]['hits']);
+
+        self::assertArrayHasKey('indexUid', $response['results'][1]);
+        self::assertArrayHasKey('hits', $response['results'][1]);
+        self::assertArrayHasKey('query', $response['results'][1]);
+        self::assertCount(4, $response['results'][1]['hits']);
+
+        $genres_seen = [];
```

Comment:
Should be 
```suggestion
        $genresSeen = [];
```
As per coding standards